Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol {
```

Comment:
No. Either call this TimerProtocol (because I don't directly implement this to get a view model)
Or extend TimedViewModelProtocol with BaseViewModelProtocol.